packages/beacon-node/src/chain/validation/attestation.ts

```diff
@@ -489,7 +488,7 @@ export function verifyPropagationSlotRange(fork: ForkName, chain: IBeaconChain,
   const earliestPermissibleSlot = Math.max(
     // slot with past tolerance of MAXIMUM_GOSSIP_CLOCK_DISPARITY_SEC
     // ATTESTATION_PROPAGATION_SLOT_RANGE = SLOTS_PER_EPOCH
-    chain.clock.slotWithPastTolerance(MAXIMUM_GOSSIP_CLOCK_DISPARITY_SEC) - SLOTS_PER_EPOCH,
+    chain.clock.slotWithFutureTolerance(MAXIMUM_GOSSIP_CLOCK_DISPARITY_MS) - SLOTS_PER_EPOCH,
```

Comment:
Why did you rename / switch these methods?

Reply:
As we really want to have future tolerance, as the peer clock may be behind.

Considering that no tests or test fixture changed, means its actually working right.